### PR TITLE
Update for VM 3

### PR DESCRIPTION
**Pull request by dzungtran**
Update integration module for Virtue Mart 3

### CODE REVIEW

File: src/paymentwall.php [hunk 1]
```diff
@@ -7,6 +7,23 @@
...
+ * 
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ * 
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
```
Comment (sevastyanovio) [inline src/paymentwall.php]: remove license text

File: src/paymentwall.php [hunk 2]
```diff
@@ -226,6 +248,15 @@ function plgVmSetOnTablePluginParamsPayment ($name, $id, &$table)
     return $this->setOnTablePluginParams($name, $id, $table);
   }
 
+    /**
+     * Addition triggers for VM3
+     * @param $data
+     * @return bool
+     */
+    function plgVmDeclarePluginParamsPaymentVM3( &$data) {
```
Comment (sevastyanovio) [inline src/paymentwall.php]: remove space before var
also you don't need pass $data by reference

Comment (dzungtran) [inline src/paymentwall.php]: - Space: ok i will remove it.
- pass $data by reference: I follow this [tuts](http://docs.virtuemart.net/tutorials/development/175-code-adjustments-for-virtuemart-3.html). Is it important?

File: src/paymentwall.php (hunk 1 above)
Comment (dzungtran) [inline src/paymentwall.php]: Okay I will remove

**Comment by sevastyanovio** [comment]
Thank you for contributing PHP SDK!

**Pull request merged**